src/wasm/kotlin/README.md

```diff
@@ -1,12 +1,56 @@
-# Kotlin --> WASM particles
-
-## Build & Test
-0. Make sure the project is [setup](../../../README.md#install)!
-1. `bazel build particles/Native/Wasm:all`
-2. `npm start`
-3. Visit [localhost:8786/shells/dev-shell/?m=https://$particles/Native/Wasm/wasm.arcs](http://localhost:8786/shells/dev-shell/?m=https://$particles/Native/Wasm/wasm.arcs)
-
-## Converting Arcs Schemas to Kotlin
-Subject to change
-1. Run `$path-to-sigh/sigh schema2wasm -k $path-from-arcs-folder-to-manifest-file -f $path-from-arcs-folder-to-output-file`
-1. The Kotlin file should be in the output file you specified above.
+# Kotlin --> Wasm particles
+
+## Setup 
+
+Our code is built by Bazel. It uses Kotlin Mutliplatform to target JVM and Wasm 
+runtimes. Bazel must be installed to build and run the code and tests, which
+reside in `src/wasm/`.
+
+1. **Installing Bazel:** run the `tools/setup` script to install Bazel on Linux
+   and MacOS (or otherwise look at that script to see which version of Bazel to 
+   download and how to set it up).
```

Comment:
Maybe point to the main setup instructions? This way we only have 1 source to update.